### ISSUE TITLE
Exclude "All Messages" folder on GMail

### ISSUE THREAD

**Issue by markusloffler** [joeyates/imap-backup#58]
I tried imap-backup and so far it works great for me. However, there is one thing: For my Gmail account, all messages are backed up twice, once for each label I sorted the messages in, and once for "All messages". Is there a way to exclude "All messages" from backup?

**Comment by joeyates** [comment]
The quick answer is to configure the backup and select which folders you want to back up. Would that be possible in your case?

If you just want to guarantee a single backup of everything, you could *only* back up "All messages". Otherwise, you could select all the *other* folders that you want to back up.

If you'd like to use the second option, but have thousands of folders, then, I agree, it would be nice to have a blacklist of specific folders to skip, but I think most people can get away with one of the two systems above.

**Comment by markusloffler** [comment]
Yes, I got many folders to backup. The problem is not primarily to enter all those in the configuration but rather to keep the configuration file updated if you add new folders. And of course, if I only backup "All messages", I loose my folder structure.

I guess I could write  a little script to create or patch the configuration file on the fly, doing a "map-backup folders" and then setup the configuration file that it includes those without the "All messages".
